acceptance/chaos_test.go

```diff
@@ -85,7 +84,7 @@ func TestChaos(t *testing.T) {
 					// These originate from DistSender when, for example, the
 					// leader is down. With more realistic retry options, we
 					// should probably not see them.
-					if _, ok := err.(*roachpb.SendError); ok || testutils.IsError(err, rpc.ErrShutdown.Error()) {
+					if _, ok := err.(*roachpb.SendError); ok || testutils.IsError(err, rpc.ErrShutdown.Error()) || testutils.IsError(err, "client is unhealthy") {
```

Comment:
maybe, but that should be in a different change. There are some things to be considered (will this `SendError` be retryable or not? If so, there are infinite retries at `DistSender`). For the purpose of the chaos test, it's convenient to get errors.